### ISSUE TITLE
Doesn't work at all with windows 10 text size is not 100% (screenshot attached)

### ISSUE THREAD

**Issue by 26medias** [ColorCop/ColorCop#13]
Hi,

I have a Surface Book which comes with a high resolution screen, which requires to set the size to 200% to really enjoy that screen.

![](https://i.imgur.com/5eodPuK.png)

However, colorcop doesn't work at all with this setting, it acts as if the screen was in 100% size, and so everything it "sees" is actually at a different location than when you point.

No way to ever select any color anymore.

**Comment by gomarion** [comment]
I have the same issue on my HP laptop Windows 10. My size is set to 125% as recommended, and there seems to be an offset when using the magnifier and even the eye dropper feature. Is there any way to fix this?

**Comment by 26medias** [comment]
Doesn't seem to be a priority for the developers.

In the meantime, you can use "Just color picker". It's not nearly as good as ColorCop, but it's a decent temporary solution until it's fixed. That's the only color picker for Windows that still works when the size is not 100%.

**Comment by gomarion** [comment]
Yeah, seeing that the last updates to this app were around 2 years ago, I don't think they'll get anything done anytime soon. Thanks for the suggestion though! Just earlier today I was able to try "Instant Eyedropper". Looks good so far. If something goes wrong with this, I'll give "Just color picker" a try. 👍

**Comment by 26medias** [comment]
_Instant Eyedropper_ is much easier to use than _Just color picker_!

I'm gonna use that now.

Thanks!

**Comment by endolith** [comment]
There's a solution in https://github.com/ColorCop/ColorCop/issues/18#issuecomment-493771026

**Comment by gomarion** [comment]
> There's a solution in [#18 (comment)](https://github.com/ColorCop/ColorCop/issues/18#issuecomment-493771026)

Oh wow! Over 2 and a half years later and I finally found a solution to this issue! Thanks a lot @endolith! I can confirm that this solution works for me!

**Comment by j4y** [comment]
Thanks everyone for your patience while we tracked this down.

The fix is now live in #262, and the app embeds the updated manifest as of commit b2ca976.

This should address the DPI/theming issues reported earlier.
